zb-debug.js

```diff
@@ -15,20 +15,20 @@ const DEBUG_FLAG = {
 
   // Use DEBUG_flow if you need to debug the flow of the program. This causes
   // prints at the beginning of many functions to print some info.
-  DEBUG_flow: false,
+  DEBUG_flow: true,
```

Comment:
These should be left as false. You can configure these at runtime by using Settings->Addons and clicking on Configure for the zigbee adapter and then entering the portion after the DEBUG_, i.e.
```
flow, frames, frameDetail
```
into the debug string would enable those 3 features.

Reply:
that was not supposed to be here.

I was just testing a new conner II I bought.

I will revert those changes....

Reply:
Sorry about that... just revert to the last changed commit.

Reply:
Yeah - that's a "feature" of github. Normally I create one branch per PR. Then it there are review changes, I just add them to the same branch. You can either add additional commits, or you can do a forced push (by editing exsiting commits).